src/com/content-item/item.js

```diff
@@ -64,19 +65,9 @@ export default class ContentItem extends Component {
 			if ( r.node && r.node.length ) {
 				var Parent = r.node[0];
 				this.setState({'parent': Parent});
-
-				return $Grade.GetMy(node.id);
 			}
 			return Promise.resolve({});
 		})
-		.then(r => {
-			if ( r.grade ) {
-				this.setState({'grade': r.grade});
-			}
-			else {
-				this.setState({'grade': []});
-			}
-		})
 		.catch(err => {
 			this.setState({ 'error': err });
```

Comment:
Prefer `({'error': err})` (note the lack of prefix and suffix spaces within the `{}`'s). A very minor change, but I'll have an explanation later.